client/src/Components/Common/Input/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import TextField from '@mui/material/TextField';
+import { makeStyles } from '@mui/styles';
+
+import PropTypes from 'prop-types';
+
+function Input({
+  label = 'inter your value',
+  widthInput = '1.5',
+  error = false,
+}) {
+  const useStyles = makeStyles({
+    input: {
+      width: `${widthInput * 200}px`,
```

Comment:
You could've simply said `width: widthInput * 200`, plain numbers are accepted and treated as pixel values.